src/BenchmarkDotNet/Jobs/PowerPlanMode.cs

```diff
@@ -6,6 +6,7 @@ namespace BenchmarkDotNet.Jobs
     public sealed class PowerPlanMode : JobMode<PowerPlanMode>
     {
         public static readonly Characteristic<PowerPlan> PowerPlanCharacteristic = CreateCharacteristic<PowerPlan>(nameof(PowerPlan));
+        public static readonly Characteristic<string> PowerPlanGuidCharacteristic = CreateCharacteristic<string>(nameof(PowerPlanGuid));
```

Comment:
Why are you using `string` instead of `Guid` here? Does `Characteristic` does not support it?

Reply:
> 
> 
> Why are you using `string` instead of `Guid` here? Does `Characteristic` does not support it?

Unfortunately, if we used Guid here, we would receive compile error. I don't have a better idea than leaving string here. In .notcs file it looks like:
As a GUID:
``` 
BenchmarkDotNet.Jobs.PowerPlanMode.PowerPlanGuidCharacteristic[job] = e9a42b02-d5df-448d-aa00-03f14749eb61;
````
As a string: 
````
BenchmarkDotNet.Jobs.PowerPlanMode.PowerPlanGuidCharacteristic[job] = $@"e9a42b02-d5df-448d-aa00-03f14749eb61";
````

Reply:
I am sorry, it looked like our templating was missing `Guid` support.

I just added it and pushed to master https://github.com/dotnet/BenchmarkDotNet/commit/04ec20b5e0c0a514e8d158684864e4f9934ae8cc

Could you please sync your branch with my change and give it one more try?